sdk/eventhub/event-hubs/src/index.ts

```diff
@@ -24,7 +25,6 @@ export { EventHubProducerClient } from "./eventHubProducerClient";
 export { SubscriptionOptions, Subscription, OptionalEventHandlers, OnErrorHandler, OnInitializeHandler, OnCloseHandler } from "./eventHubConsumerClientModels";
 export { EventPosition } from "./eventPosition";
 export { PartitionProperties, EventHubProperties } from "./managementClient";
-export { EventHubProducer } from "./sender";
 export { EventIteratorOptions } from "./receiver";
```

Comment:
I think we can remove some things from this file.

Do we need to export:
- PartitionLoadBalancer
- PartitionProcessor
- EventIteratorOptions
- EventHubClientOptions (maybe if we extend this)

Reply:
Options is still being exported but I was able to cleanly get rid of the other three. I also got rid of a few more as I found them. 

We'll need to do a more thorough sweep.